qutip/qobj.py

```diff
@@ -1137,31 +1131,32 @@ def diag(self):
             return np.real(out)
 
     def expm(self, method='dense'):
-        """Matrix exponential of quantum operator.
-
-        Input operator must be square.
+        """Matrix exponential of a square quantum operator.
 
         Parameters
         ----------
         method : str {'dense', 'sparse'}
             Use set method to use to calculate the matrix exponentiation. The
             available choices includes 'dense' and 'sparse'.  Since the
             exponential of a matrix is nearly always dense, method='dense'
-            is set as default.s
+            is set as default.
 
         Returns
         -------
-        oper : :class:`qutip.Qobj`
+        :obj:`.Qobj`
             Exponentiated quantum operator.
 
         Raises
         ------
-        TypeError
-            Quantum operator is not square.
+        TypeError : Invalid operand for matrix exponential
+            When dimensions do not match i.e. input is not a square operator.
+
+        ValueError : Method must be 'dense' or 'sparse'
+            When available deafult method is not chosen.
```

Comment:
don't copy error messages, also the ValueError is an "obvious" error because the allowed values are specified